parser_actions.go

```diff
@@ -95,8 +95,10 @@ func (ap *AnsiParser) csiDispatch() error {
 }
 
 func (ap *AnsiParser) print() error {
-	logger.Infof("AnsiParser::print %#x", ap.context.currentChar)
```

Comment:
Actually, the core problem is that we're emitting one character at a time. If we emitted a sequence, the logging calls would not likely matter. (Which was the fix Gabriel had intended, by the way.) If emitting a sequence is too much work at this time, then elide the calls for now.